web/src/views/HomeView/SingleStat.vue

```diff
@@ -18,7 +18,14 @@
         class="pa-0"
         data-id="name"
       >
-        {{ name }}
+        <a
+          v-if="href"
+          :href="href"
+        >
+          {{ name }}</a>
```

Comment:
```suggestion
          {{ name }}
        </a>
```

Reply:
Can't do it. The linebreak is interpreted by the html renderer as a space, which makes the link look ugly (the underline for hyperlinks extends one space to the right).

Reply:
Oh, alright then carry on.